### DISCUSSION TITLE
Bring back Extended Validation certificate indicator in the address bar

### DISCUSSION THREAD

**Discussion by jobbautista9** [Floorp-Projects/Floorp#1252] [Feature Request]
Disabled in Firefox 70: [bug 1572936](https://bugzilla.mozilla.org/show_bug.cgi?id=1572936)

Removed entirely in Firefox 77: [bug 1599729](https://bugzilla.mozilla.org/show_bug.cgi?id=1599729)

In my opinion, the removal of this useful indicator is a mistake, especially in a time where it is stupidly easy to create a phishing site and slap an HTTPS on it to show it is "legitimate". This still has not changed despite what the mainstream wants us to believe, and an EV indicator would still be useful for power users who know what their browser looks like.

**Comment by Ryosuke-Asano** [comment]
HMM
